q2_vsearch/plugin_setup.py

```diff
@@ -404,12 +406,21 @@
                   'abundances).'),
     },
     parameter_descriptions={
+        'method': ('Which algorithm to use.'),
```

Comment:
Can we add some text to briefly explain the differences? If the differences are too technical/subtle to explain here we should at least point to the official documentation.